example/alertmanager-config.yaml

```diff
@@ -0,0 +1,18 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: alertmanager-main
+data:
+  alertmanager.yaml: |-
+    global:
+      resolve_timeout: 5m
+    route:
+      group_by: ['alertname']
```

Comment:
If I understand correctly that would mean that different alerts belonging to the same job would be sent out as a notification, correct?